### ISSUE TITLE
how do you render block outlines?

### ISSUE THREAD

**Issue by JanCantCode** [19MisterX98/SeedcrackerX#176]
Not sure if this even belongs  here, but ive been trying to realize how you can render blocks / block outlines using fabric, and ive allways failed lmao. any help?

**Comment by JanCantCode** [comment]
yes, i have seen https://github.com/19MisterX98/SeedcrackerX/tree/master/src/main/java/kaptainwutax/seedcrackerX/render but i dont reall see any rendering done in there

**Comment by notpoiu** [comment]
This code is used to render something called the outlines. The code checks the rendering setting from a configuration (Config.get().render), and if it's set to "OFF", the method returns immediately. The method uses a matrix stack and a camera to calculate the position of the player camera in the game world, and then uses the Tessellator and BufferBuilder to define and render lines with specific properties (width, blend mode, etc.) The method then renders each finder in the "finderControl" list if it should be rendered, and finally draws the lines and cleans up the rendering state.

**Comment by 19MisterX98** [comment]
Rendering in mc is messy and you probably shouldnt look at my code cause I am a noob in that kind of stuff aswell
